app/controllers/comments_controller.rb

```diff
@@ -50,6 +52,11 @@ def create
     if comment.valid? && params[:preview].blank? && comment.save
       comment.current_vote = {vote: 1}
       render_created_comment(comment)
+
+      if comment.parent_comment
+        old_reply_count = comment.parent_comment.reply_count
+        comment.parent_comment.update_column(reply_count: old_reply_count + 1)
```

Comment:
I'd like to use [counter_cache](https://api.rubyonrails.org/classes/ActiveRecord/Calculations.html#method-i-async_count) for this if possible. If not, I would rather see something like `comment.parent_comment.update_column(reply_count: comment.parent_comment.replies.not.deleted.count))` (I'm guessing at the scope name for non-deleted comments)  to have the database count rather than try to keep `+ 1` and `- 1` updates correct.